code/modules/clothing/outfits/ert.dm

```diff
@@ -62,6 +62,41 @@
 	R.keyslot = new /obj/item/encryptionkey/heads/captain
 	R.recalculateChannels()
 
+/datum/outfit/ert/mining
+	name = "A Dwarven Miner"
+
+	id = /obj/item/card/id/ert
+	suit = /obj/item/clothing/suit/space/hardsuit/ert/paranormal/beserker
+	suit_store = /obj/item/tank/internals/oxygen/tactical
+	r_hand = /obj/item/kinetic_crusher/mega
+	glasses = /obj/item/clothing/glasses/hud/health/meson
+	gloves = /obj/item/clothing/gloves/gauntlets
+	back = /obj/item/storage/backpack/explorer
+	belt = /obj/item/storage/belt/mining
+	mask = /obj/item/clothing/mask/gas/explorer
+	shoes = /obj/item/clothing/shoes/bhop
+	uniform = /obj/item/clothing/under/rank/miner/lavaland
+	backpack_contents = list(
+		/obj/item/storage/box/survival_mining=1,
+		/obj/item/crusher_trophy/demon_claws=1,
+		/obj/item/crusher_trophy/watcher_wing=1,
+		/obj/item/reagent_containers/autoinjector/medipen/survival=3,
+		/obj/item/kinetic_javelin=1,
+		/obj/item/dnainjector/dwarf=1,
```

Comment:
rather than giving them an injector, force the mutation on them directly in post_equip